.circleci/config.yml

```diff
@@ -1,11 +1,25 @@
 version: 2.1
 orbs:
   maven: circleci/maven@1.4.0
+
+executors:
+  jdk8:
+    docker:
+      - image: cimg/openjdk:8.0
```

Comment:
If we are going to support jdk8 (I think we can move to JDK11, though), adding `--release 8` javac option will be helpful for enforcing JDK8 support

Reply:
Thanks for your advice. I will consider adding it when bumping the minimum JDK to 11.